.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
Great job😊 at implementing the linter configuration, but I see that there is an error🐛 at the stylelint linter, so please fix this linter by adding this command at the terminal
 npx stylelint "**/*.{css,scss}" --fix , so the linter will be happy for that 😃


![review](https://user-images.githubusercontent.com/116716918/214054085-5ad81a79-e0a5-4a94-bec6-060c45eb7246.png)

Reply:
Well done on this one 💯 keep it up. 🙂